template/page-impact.html.jinja

```diff
@@ -8,119 +8,226 @@
-              <p class="stat-highlight h6 ml-10 mb-0">
-                <img src="{{currentpage.prefix}}assets/img/icons/green-up-arrow.svg">
-                {% trans %}+21.4%{% endtrans %}
-              </p>
-            </div>
-            <p class="numbers">{% trans %}$10.5T{% endtrans %}</p>
-          </div>
-        </div>
+        <h6 class="eyebrow mb-3">A sustainable future</h6>
+        <h2 class="h4 h2-sm mb-8">{% trans %}What makes the XRPL sustainable?{% endtrans %}</h2>
+        <h5 class="longform mb-10">{% trans %}XRPL’s unique consensus mechanism is eco-friendly—it does not require mining to settle transactions. This results in efficiency without sacrificing security, decentralization, or scalability.{% endtrans %}</h5>
+        <p>The trivial amount of energy it does consume is then neutralized with carbon credits through EW Zero, an open-source blockchain decarbonization tool.</p>
       </div>
+      
     </div>
   </section>
 
-  <div class="position-relative d-none-sm">
-    <img src="./img/backgrounds/impact-magenta.svg" id="impact-magenta">
-  </div>
-
+  <!-- Card -->
   <section class="container-new py-26">
-    <div class="col-md-6 offset-md-3 p-10-until-sm pl-0-sm pr-0-sm">
-      <h2 class="h4 h2-sm mb-8">{% trans %}Are All Digital Assets Alike?{% endtrans %}</h2>
-      <h5 class="longform mb-10">{% trans %}All digital assets have different strengths that make them ideal for various use cases.{% endtrans %}</h5>
-      <p class="mb-6">{% trans %}Bitcoin has historically been recognized as a store of value and Ether (ETH) for its smart contract capabilities.{% endtrans %}</p>
-      <p class="mb-6">{% trans %}XRP was designed to be optimal for global payments—it’s fast, cheap, scalable and energy-efficient. But the XRP Ledger is also increasingly popular for innovative blockchain use cases, like NFTs and DeFi.{% endtrans %}</p>
-      <p class="mb-6">{% trans %}The same characteristics that make XRP ideal for global transactions mean it’s also better for our environment. XRP is green by nature.{% endtrans %}</p>
+    <div class="col-md-6 offset-md-3 p-6-sm p-10-until-sm br-8 cta-card">
+      
+      <img src="./img/backgrounds/cta-community-purple.svg" class="cta cta-top-left">
+      <img src="./img/backgrounds/cta-calculator-green.svg" class="cta cta-bottom-right">
+      <div class="z-index-1 position-relative">
+        <div class="d-flex flex-column-reverse">
+          <h2 class="h4 h2-sm mb-10-until-sm mb-8-sm">{% trans %}Featured companies & projects running on the XRP Ledger.{% endtrans %}</h2>
+          <h6 class="eyebrow mb-3">{% trans %}Sustainable Projects{% endtrans %}</h6>
+        </div>
+        <p class="mb-10">{% trans %}Learn more about companies and developers who are using the XRP Ledger to solve interesting problems efficiently and sustainably.{% endtrans %}</p>
+        <a class="btn btn-primary btn-arrow" href="/">{% trans %}See More{% endtrans %}</a>
```

Comment:
I think `/` is not the intended href here? Maybe you meant to link the new Use Cases page? (If so, don't forget to include `{{currentpage.prefix}}` in the URL so it works in the Japanese target.)

Reply:
changed to link to usecases.